src/public/script.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable */
+const messageForm = document.getElementById("send-container");
+const messageInput = document.getElementById("message-input");
+const messageContainer = document.getElementById("message-container");
+const userName = prompt("What is your name?");
```

Comment:
You don't need to prompt the user for their names, get all this info from their login session.